src/components/dialog-full-screen/notes.md

```diff
@@ -0,0 +1,8 @@
+# Designer Notes
+- Useful to perform an action in context without navigating the user to a separate page.
+- Whilst a standard Dialog component’s width might constrain what you can do, the full screen dialog uses the full width and height available.
+- A good example could be importing a large volume of data, and checking for errors, in the context of an underlying Table of data.
+
+# Related Components
+- Simple task in context? [Try Dialog](/components/dialog "Try Dialog").
```

Comment:
Can we stay with format `[Try Dialog](/components/dialog "Dialog")`?

Reply:
I can do, but I was followed the convention used in previously Storybook'ed components. See `Alert`, `Button` and `Button Toggle`. 

If this is required, I can update all of the existing `notes.md` files?

Reply:
It would be nice. I missed it. Thanks